pkg/interop/interop_util_test.go

```diff
@@ -0,0 +1,92 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package interop_test
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+
+	"go.thethings.network/lorawan-stack/pkg/util/test"
+)
+
+var (
+	WorkingDirectory = test.Must(os.Getwd()).(string)
```

Comment:
Why do we need absolute paths?

Reply:
They work always, regardless of the working directory

Reply:
Isn't the point of package tests that they are self-containing? This is also the reason for `testdata` and ensuring that test functionality doesn't end up in binaries.

Also, if you `go test pkg/interop`, the working directory + `/testdata` doesn't work, does it?

Reply:
Client tests write configuration to a temporary directory and use that directory as the repository. `testdata` is not present there.
Of course, we could copy the `testdata` contents to the temporary directory as part of setup, but IMO it's much simpler and cleaner to just use the absolute path to the certificates.
Note, that in the test cases operation with relative paths is tested as well.

Reply:
The tests are self-containing and `go test ./pkg/interop` works as expected